docs/topics/signals.rst

```diff
@@ -295,6 +295,23 @@ request_reached_downloader
     :param spider: the spider that yielded the request
     :type spider: :class:`~scrapy.spiders.Spider` object
 
+request_left_downloader
+---------------------------
+
+.. signal:: request_left_downloader
+.. function:: request_left_downloader(request, spider)
+
+    Sent when a :class:`~scrapy.http.Request` left downloader even in case of
+    failure.
+
+    The signal does not support returning deferreds from their handlers.
```

Comment:
```suggestion
    This signal does not support returning deferreds from its handlers.
```
I'm hopping to get the other occurrences of "their" updated in #4205

Reply:
It is done in accord with description of other signals, e.g https://github.com/scrapy/scrapy/blame/master/docs/topics/signals.rst#L257

Reply:
I'm no native English speaker, but I believe "this" sounds better than "the". That's probably subjective though.
However I'm pretty sure "their" is incorrect, since we're talking about one signal, not many.